sandbox/OpenIddict.Sandbox.AspNet.Server/Controllers/AuthorizationController.cs

```diff
@@ -57,7 +62,7 @@ public async Task<ActionResult> Authorize()
             // If the user principal can't be extracted or the cookie is too old, redirect the user to the login page.
             var result = await context.Authentication.AuthenticateAsync(DefaultAuthenticationTypes.ApplicationCookie);
             if (result?.Identity == null || (request.MaxAge != null && result.Properties?.IssuedUtc != null &&
-                DateTimeOffset.UtcNow - result.Properties.IssuedUtc > TimeSpan.FromSeconds(request.MaxAge.Value)))
+                _openIddictCoreOptions.Value.GetUtcNow() - result.Properties.IssuedUtc > TimeSpan.FromSeconds(request.MaxAge.Value)))
```

Comment:
Let's keep `DateTimeOffset.UtcNow` in the samples, there's no need for extensibility there.